data/strings/strings.txt

```diff
@@ -29890,49 +29890,142 @@
+    sv = Organic Maps behöver platsåtkomst. När det är säkert kan du kontrollera aviseringen på din telefon.
+    sw = Organic Maps zinahitaji ufikiaji wa eneo. Ikiwa salama, angalia arifa kwenye simu yako.
+    th = แผนที่ทั่วไปจำเป็นต้องเข้าถึงตำแหน่ง เมื่อปลอดภัยแล้ว ให้ตรวจสอบการแจ้งเตือนบนโทรศัพท์ของคุณ
+    tr = Organic Maps konum erişimine ihtiyacı vardır. Güvenli olduğunda, telefonunuzdaki bildirimi kontrol edin.
+    uk = Для роботи Organic Maps потрібен доступ до місцезнаходження. Коли це буде безпечно, перевірте сповіщення на телефоні.
+    vi = Bản đồ không phải trả tiền cần quyền truy cập vị trí. Khi an toàn, hãy kiểm tra thông báo trên điện thoại của bạn.
+    zh-Hans = 有机地图需要位置访问权限。安全后，查看手机上的通知。
+    zh-Hant = 有機地圖需要位置存取權限。當安全時，請查看手機上的通知。
+
+  [aa_request_permission_notification]
+    comment = Notification title for permission request from AA.
+    tags = android
+    en = This app needs your permission
+    af = Hierdie toepassing benodig jou toestemming
+    ar = هذا التطبيق يحتاج إلى إذنك
+    be = Гэтай праграме патрабуецца ваш дазвол
+    bg = Това приложение се нуждае от вашето разрешение
+    ca = Aquesta aplicació necessita el vostre permís
+    cs = Tato aplikace potřebuje vaše povolení
+    da = Denne app har brug for din tilladelse
+    de = Diese App braucht deine Erlaubnis
+    el = Αυτή η εφαρμογή χρειάζεται την άδειά σας
+    es = Esta aplicación necesita tu permiso
+    et = See rakendus vajab teie luba
+    eu = Aplikazio honek zure baimena behar du
+    fa = این برنامه به اجازه شما نیاز دارد
+    fi = Tämä sovellus tarvitsee lupasi
+    fr = Cette application a besoin de ta permission
+    he = אפליקציה זו זקוקה לאישור שלך
+    hi = इस ऐप को आपकी अनुमति की आवश्यकता है
+    hu = Ennek az alkalmazásnak szüksége van az Ön engedélyére
+    id = Aplikasi ini membutuhkan izin Anda
+    it = Questa applicazione ha bisogno del tuo permesso
+    ja = このアプリはあなたの許可を必要とする
+    ko = 이 앱은 사용자의 승인이 필요합니다.
+    lt = Šiai programėlei reikia jūsų leidimo
+    mr = या अॅपला तुमची परवानगी आवश्यक आहे
+    nb = Denne appen trenger din tillatelse
+    nl = Deze app heeft je toestemming nodig
+    pl = Ta aplikacja wymaga Twojej zgody
+    pt = Esta aplicação precisa da tua autorização
+    pt-BR = Este aplicativo precisa da sua permissão
+    ro = Această aplicație are nevoie de permisiunea dvs.
+    ru = Этому приложению нужно ваше разрешение
+    sk = Táto aplikácia potrebuje vaše povolenie
+    sv = Denna app behöver ditt tillstånd
+    sw = Programu hii inahitaji idhini yako
+    th = แอพนี้ต้องได้รับอนุญาตจากคุณ
+    tr = Bu uygulamanın izninize ihtiyacı var
+    uk = Ця програма потребує вашого дозволу
+    vi = Ứng dụng này cần sự cho phép của bạn
+    zh-Hans = 此应用程序需要您的许可
+    zh-Hant = 此應用程式需要您的許可
+
+  [aa_request_permission_activity_text]
+    comment = The text in the activity for location permission request.
+    tags = android
+    en = Organic Maps in Android Auto needs a location permission to work effectively
```

Comment:
1. Would `requires` be better than `needs`?
2. Is working "inefficiently" possible without permission?
```suggestion
    en = Organic Maps in Android Auto requires location permission to work
```